src/sentry/models/dashboard_widget.py

```diff
@@ -45,11 +45,13 @@ def get_id_for_type_name(cls, type_name):
 class DashboardWidgetTypes(TypesClass):
     DISCOVER = 0
     ISSUE = 1
-    METRICS = 2
+    RELEASE_HEALTH = 2
+    METRICS = 3
     TYPES = [
         (DISCOVER, "discover"),
         (ISSUE, "issue"),
-        (METRICS, "metrics"),
+        (RELEASE_HEALTH, "metrics"),
+        (METRICS, "custom-metrics"),
```

Comment:
Let's maybe leave a comment similar to this one?
https://github.com/getsentry/sentry/blob/ogi/feat/render-ddm-widget/static/app/views/dashboards/types.tsx#L27-L28